lib/cisco_node_utils/radius_server.rb

```diff
@@ -39,6 +39,24 @@ def initialize(name, instantiate=true)
       end
       @name = name
 
+      if platform == :ios_xr
```

Comment:
@jonnytpuppet Did you end up changing the method to use the provided `auth_p` and `acct_p` values on Nexus?  It looks like you have changed it, but the diffs do not make it clear